### ISSUE TITLE
tests/docker-scripts broken in docker compose v2.24.6

### ISSUE THREAD

**Issue by vbabich** [deephaven/web-client-ui#1891]
```
docker --version
Docker version 25.0.3, build 4debf41

Docker Compose version v2.24.6
```

```
npm run e2e:docker

> @deephaven/web-client-ui@0.0.1 e2e:docker
> ./tests/docker-scripts/run.sh web-ui-tests

service "web-ui-tests" can't be used with `extends` as it declare `depends_on`
service "web-ui-tests" can't be used with `extends` as it declare `depends_on`
```

```
npm run e2e:update-ci-snapshots

> @deephaven/web-client-ui@0.0.1 e2e:update-ci-snapshots
> ./tests/docker-scripts/run.sh web-ui-update-snapshots

service "web-ui-tests" can't be used with `extends` as it declare `depends_on`
service "web-ui-tests" can't be used with `extends` as it declare `depends_on`
```

Looks like there was a change in docker compose v2.24.6 that breaks our e2e scripts - https://github.com/docker/compose/issues/11544

**Comment by vbabich** [comment]
Should be fixed in docker compose v2.24.7

**Comment by vbabich** [comment]
Latest docker desktop includes docker compose v2.24.6. Can use this as a temporary fix until docker desktop with updated compose is published: 
```version: '3.8'

services:
  dhc-server:
    image: ghcr.io/deephaven/server:${DHC_VERSION:-edge}
    pull_policy: always
    volumes:
      - ./data:/data
    expose:
      - 10000
    environment:
      - START_OPTS=-Xmx4g -DAuthHandlers=io.deephaven.auth.AnonymousAuthenticationHandler -Ddeephaven.console.type=python -Ddeephaven.application.dir=./data/app.d

  web-ui-tests: &web-ui-tests
    build:
      context: ../../
      dockerfile: ./tests/docker-scripts/Dockerfile
    ipc: host
    environment:
      - VITE_PROXY_URL=http://dhc-server:10000
    ports:
      - 9323:9323
    volumes:
      - ../../tests:/work/tests
      - ../../test-results:/work/test-results
      - ../../playwright-report:/work/playwright-report
    entrypoint: 'npm run e2e -- --config=playwright-ci.config.ts'
    depends_on:
      dhc-server:
        condition: service_healthy

  web-ui-update-snapshots:
    <<: *web-ui-tests
    entrypoint: 'npm run e2e:update-snapshots -- --config=playwright-ci.config.ts'
```
